module/Calibrator.h

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file Calibrator.h
+ * @brief キャリブレーションからスタートまでを担当するクラス
+ * @author aridome222
+ */
+
+#ifndef CALIBRATOR_H
+#define CALIBRATOR_H
+
+#include "Measurer.h"
+#include "Timer.h"
+
+class Calibrator {
+ public:
+  /**
+   * コンストラクタ
+   */
+  Calibrator();
+
+  /**
+   * キャリブレーション処理（入力系）をまとめて実行する
+   */
+  void run();
+
+  /**
+   * スタート合図が出るまで待機状態にする
+   */
+  void waitForStart();
+
+  /**
+   * isLeftCourseのゲッター
+   * @return true:Lコース, false:Rコース
+   */
+  bool getIsLeftCourse();
+
+  /**
+   * targetBrightnessのゲッター
+   * @return 目標輝度
+   */
+  int getTargetBrightness();
+
+ private:
+  bool isLeftCourse;     // true:左コース, false: 右コース
```

Comment:
```suggestion
  bool isLeftCourse;     // true:Lコース, false: Rコース
```
コメントで動作に影響ないうえに細かいですが。getLefftCourseと書き方合わせるといいと思います。
それかgetIsLeftCourseの法を右コース、左コースに合わせてもいいと思いますが、後の記述がLRなのでこの修正がいいかもしれません。